base_upflow/tests/test_account_move.py

```diff
@@ -0,0 +1,50 @@
+from odoo.tests.common import SavepointCase
+
+class TestAccountMove(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner = cls.env["res.partner"].create(
+            {"name": "Test partner", "email": "email@example.co"}
+        )
+        cls.journal = cls.env["account.journal"].create(
+            {"name": "Test journal", "type": "sale", "code": "TEST"}
+        )
+        cls.product = cls.env["product.product"].create(
+            {"name": "Test product", "type": "service"}
+        )
+        cls.account = cls.env["account.account"].create(
+            {"name": "Test account", "code": "TEST", "user_type_id": 1, "reconcile": True}
```

Comment:
instead of hardcoded type id is there a xmlid we could use or using the result of a search to avoid direct dependency to `l10n_generic_coa`?

Reply:
I've created a new account.account.type instead of passing hardcoded, should it be better ?

Reply:
should be fine ! let's see if OCA build is happy with